NonWeb.md

```diff
@@ -20,8 +21,19 @@ both browsers and node.js. This situation is expected to be first encountered
 with POSIX features such as file I/O. In that respect, WebAssembly would err
 towards standardizing existing practice through libraries, and let developers
 choose which libraries to use.
+Similarly, the WebGL and SDL interface (
+[exposed by emscripten](http://kripken.github.io/emscripten-site/docs/porting/multimedia_and_graphics/OpenGL-support.html)
+in C/C++) could be made into a shared spec with non-browser mobile/desktop
+environments, allowing for cross platform browser & non-browser games and other
+things.
```

Comment:
SDL is a fairly clean and simple C style API, so that would be the preferred way, but the low level approach would work too (which presumably is what emscripten uses today).